### DISCUSSION TITLE
Node title going behind path

### DISCUSSION THREAD

**Discussion by Rajgupta7080** [joyceworks/flowchart-react#40] [Q&A]
I am trying to giving title of node but it is going behind the path I am trying to increase z-index but it is not working
![image](https://github.com/joyceworks/flowchart-react/assets/107268454/36380eb4-c42b-4093-855a-885994af5fe0)

**Comment by iamppz** [comment]
Thank you for the feedback, I will fix it later.

**Comment by Rajgupta7080** [comment]
Thank You @iamppz

**Comment by iamppz** [comment]
Hi, you can set connectionPosition to "bottom" to make the connection line behind the node:
``` javascript
<flowchart connectionPosition="bottom" ... />
```
